README.md

```diff
@@ -414,3 +414,36 @@ anchore:
   dockerfile: ""
 
 ```
+
+### Attaching an SBOM to an image as an attestation
```

Comment:
One small consideration: since "attach" and "attest" have two similar but distinct meanings within the cosign ecosystem, we may want to avoid mixing those terms when describing a single functionality, to help avoid confusion for users.